codalab/apps/chahub/utils.py

```diff
@@ -0,0 +1,74 @@
+from django.conf import settings
+
+import logging
+import os
+import requests
+import json
+
+logger = logging.getLogger(__name__)
+
+
+def send_to_chahub(endpoint, raw_data):
+    """
+    Does a post request to the specified API endpoint on chahub with the inputted data.
+    :param endpoint: String designating which API endpoint; IE: 'producers/'
+    :param raw_data: Dictionary containing data we are sending away to the endpoint.
+    :return:
+    """
+    try:
+        _chahub_api_url_check()
+    except KeyError:
+        logger.info("CHAHUB_API_URL not defined in settings!")
+        return
+
+    chahub_online = _chahub_online_check()
+
+    if not chahub_online:
+        logger.info("Chahub is currently offline. Cancelling.")
+        raise requests.ConnectionError("Chahub is currently offline. Please try again later.")
+
+    url = "{}{}".format(settings.CHAHUB_API_URL, endpoint)
+
+    data = json.dumps(raw_data)
+
+    try:
+        _chahub_send_data(url, data)
+    except requests.ConnectionError:
+        logger.info("Unable to POST data to Chahub. There was an error, please try again.")
+
+
+def _chahub_api_url_check():
```

Comment:
I think we should just use `assert yadda yadda` instead of this function, now that I see it again?